aries-site/src/pages/components/button.mdx

```diff
@@ -82,6 +83,23 @@ Secondary buttons are used to represent calls to action that are either in suppo
   <SecondaryButtonExample />
 </Example>
 
+### Toolbar Button
```

Comment:
Wondering about order of placement on the page... is Default button getting pushed too low considering it is a "default"?

I guess default is the very first example on the page, so talking myself out of this comment.